docs/whats-new/staging-release-notes.md

```diff
@@ -1,16 +1,39 @@
 # Staging release notes
 
+## Staging release 31 August 2022
+Frontend version: <version number in the environment> | Backend version: <version number in the environment>
+
+**New features** - **Backend**
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Terminate accounts that are inactive for 30 days from creation</summary>
```

Comment:
terminating accounts that are inactive for 30 days is not the new feature, the new feature is actually sending the notification emails 7 days prior to termination until the actual termination, so a total of 7 days

Reply:
got it, but the email sent to user does not clearly state that info.